app/lib/features/chat/widgets/custom_input.dart

```diff
@@ -217,19 +218,14 @@ class __ChatInputState extends ConsumerState<_ChatInput> {
     }
   }
 
-  @override
-  void didChangeDependencies() {
-    super.didChangeDependencies();
-    _setController();
-  }
-
   void _setController() {
+    // putting constant colors here as context isn't accessible in initState()
     final triggerStyles = {
       '@': TextStyle(
-        color: Theme.of(context).colorScheme.onSecondary,
+        color: Colors.white,
         height: 0.5,
         background: Paint()
-          ..color = Theme.of(context).colorScheme.secondary
+          ..color = const Color(0xFF74A64D)
```

Comment:
oh, noes... static color ... I see the almighty looking down disapprovingly to us ... I can feel his pain and sorrow in this sight ...

Reply:
Sorry, but it's just how the framework works. We can't have ```BuildContext``` before widget build and we need to set it once only in ```initState()```.